docker-compose.yml

```diff
@@ -69,3 +69,7 @@ services:
       - BUCKET_1_PER_SECOND=5
     ports:
       - "127.0.0.1:9231:9231"
+  gpubsub:
+    image: kinok/google-pubsub-emulator
```

Comment:
Not saying to change this, but just out of curiosity since it looks like there are a lot of Docker images for the emulator, why this one specifically?

Reply:
This one seemed the most up-to-date.